src/WPF/WPF.Viewer/Samples/UtilityNetwork/CreateLoadReport/CreateLoadReport.xaml

```diff
@@ -0,0 +1,158 @@
+﻿<UserControl x:Class="ArcGIS.WPF.Samples.CreateLoadReport.CreateLoadReport"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="0.5*" />
+            <ColumnDefinition />
+        </Grid.ColumnDefinitions>
+        <StackPanel Grid.Column="0" Background="LightGray">
+            <TextBlock FontSize="20"
+                       Text="1. Select a phase type using the dropdown"
```

Comment:
I think "Add phases from the dropdown list, then run report." works here as well.